### ISSUE TITLE
@ mentioning a user does not create a link to user's profile

### ISSUE THREAD

**Issue by meaghanlewis** [github/VisualStudio#1164]
__Version__
- GitHub Extension for Visual Studio  version: 2.3.0.20
- Visual Studio version: 2017

__What happened__ 
@ user does not create a clickable link to user's profile in VS as it does in .com

<img width="838" alt="screen shot 2017-08-18 at 1 35 10 pm" src="https://user-images.githubusercontent.com/10103121/29477141-ea3de1b8-841b-11e7-9a83-6f46d0b592fb.png">

**Comment by grokys** [comment]
Yes, we plan to address this soon - it will require a new feature adding to our markdown parser which is why it's not been done yet,
